lsp-methods.el

```diff
@@ -39,6 +39,7 @@
   (get-root nil :read-only t)
   (ignore-regexps nil :read-only t)
   (ignore-messages nil :read-only t)
+  (prefix-function nil :read-only t)
```

Comment:
Could you make `prefix-function` the last struct member? Adding a new member in the middle of the struct breaks binary compatibilities with byte-compiled client packages.

Reply:
Done